jekyll/_cci2/runner-config-reference.adoc

```diff
@@ -53,7 +53,7 @@ api:
 === runner.name
 `$LAUNCH_AGENT_RUNNER_NAME`
 
-`RUNNER_NAME` is a unique name assigned to this particular machine runner. CircleCI recommends using the hostname of the machine so that it can be used to identify the agent when viewing statuses and job results in the UI on the CircleCI web app.
+`RUNNER_NAME` is a unique name of your choosing assigned to this particular machine runner. CircleCI recommends using the hostname of the machine so that it can be used to identify the agent when viewing statuses and job results in the UI on the CircleCI web app.
```

Comment:
Wow, I did not figure this out at all. I think it's important to note that `RUNNER_NAME` cannot include the `/` character. I found this out the hard way when it failed to start up. It would be helpful to know if there are any other characters that can't be in `RUNNER_NAME`.

Reply:
@lukaso oh interesting, I don't know think I knew that either! I'll double check with the team that works on this, thanks.

Reply:
Hey @lukaso, do you remember if the error you got was `name not valid` (or what the error was)? 

The team says it would make sense `/` would throw an error since we use that on the back end to separate name space and resource classes so it would be a protected special character. Aside from letters and numbers, it would appear that `. () - _` are the only other characters allowed. We can definitely call this out in the docs.

Reply:
I don't remember the error and my terminal log on the runner doesn't go back far enough.

I do remember I had to run the agent manually, because it failed/stopped before creating the log file, so it was impossible to figure out what was going wrong using the standard launchctl. If the team could get logging up and running before crashing out on a bad config, that would help a lot.

Reply:
@lukaso thanks, I will pass that on!

Reply:
Hey @lukaso the team had a suggestion for the logs - you should be able to edit the plist file and provide the path to your logs for errors (we have the example to copy in the docs https://circleci.com/docs/runner-installation-mac/#create-and-install-the-launchd-plist). On line 40 you can see
```
        <key>StandardErrorPath</key>
        <string>/dev/null</string>
```
and you should be able to replace `/dev/null` with the path to your log file. They're going to look into trying to make this a default, thanks!